mcu_proxy.go

```diff
@@ -269,6 +278,8 @@ func (s *mcuProxySubscriber) ProcessEvent(msg *EventProxyServerMessage) {
 	switch msg.Type {
 	case "ice-completed":
 		s.listener.OnIceCompleted(s)
+	case "subscriber-sid-updated":
+		s.sid = msg.Sid
```

Comment:
```suggestion
		s.sid = msg.Sid
		s.listener.SubscriberSidUpdated(s)
```

Reply:
Iinitially I did not add it because `SubscriberSidUpdated` was currently empty in _clientsession.go_. Nevertheless I agree that it is better to have it already prepared in case that changes in the future :-)